contrib/kind.sh

```diff
@@ -531,12 +551,27 @@ set_default_params() {
   JOIN_SUBNET_IPV4=${JOIN_SUBNET_IPV4:-100.64.0.0/16}
   JOIN_SUBNET_IPV6=${JOIN_SUBNET_IPV6:-fd98::/64}
   KIND_NUM_MASTER=1
+  OVN_INTERCONNECT_ENABLE=${OVN_INTERCONNECT_ENABLE:-false}
+  KIND_NUM_NODES_PER_ZONE=${KIND_NUM_NODES_PER_ZONE:-2}
   if [ "$OVN_HA" == true ]; then
     KIND_NUM_MASTER=3
-    KIND_NUM_WORKER=${KIND_NUM_WORKER:-0}
+    NUM_WORKER=${KIND_NUM_WORKER:-0}
+    # Disabled interconnect if HA is enabled (for now)
+    OVN_INTERCONNECT_ENABLE="false"
   else
-    KIND_NUM_WORKER=${KIND_NUM_WORKER:-2}
+    NUM_WORKER=${KIND_NUM_WORKER:-2}
   fi
+
+  if [ "$OVN_INTERCONNECT_ENABLE" == false ]; then
+    KIND_NUM_ADDITIONAL_ZONES="0"
```

Comment:
Sorry, but one more observation on this specific change.

Not sure if it would be possible, but it would be nice to keep the default number of workers as 3, independent on whether interconnect is used or not. Right now we are creating 4 workers with ovn-ic and 3 when using non-ovn-ic clusters. On the other hand, I can see that `KIND_NUM_NODES_PER_ZONE=2`, so we end up with 2 zones and each has a node w/out db. So maybe it is okay and we just keep it this way for sake of simplicity. :)

```
❯ k get nodes
NAME                STATUS   ROLES           AGE   VERSION
ovn-control-plane   Ready    control-plane   16m   v1.26.0
ovn-worker          Ready    control-plane   16m   v1.26.0
ovn-worker2         Ready    <none>          16m   v1.26.0
ovn-worker3         Ready    control-plane   16m   v1.26.0
ovn-worker4         Ready    <none>          16m   v1.26.0
```

```
...
+ OVN_INTERCONNECT_ENABLE=true
+ KIND_NUM_NODES_PER_ZONE=2
+ '[' false == true ']'
+ NUM_WORKER=2
+ '[' true == false ']'
+ KIND_NUM_ADDITIONAL_ZONES=2
+ NUM_WORKER=0
+ KIND_NUM_WORKER=0
+ KIND_TOTAL_NUM_WORKER=4
```